src/app/cli/src/init/client.ml

```diff
@@ -1022,7 +1022,7 @@ let hash_ledger =
          | Error err ->
              Format.eprintf "Could not parse JSON in file %s: %s@" ledger_file
                err ;
-             ignore (exit 1))
+             ignore (exit 1 : 'a))
```

Comment:
Can't we just return `exit 1` since it's result is polymorphic already (same as `ignore`)?

Reply:
Good catch! Done :)